### PR TITLE
Feature/fix network

### PR DESCRIPTION
**Pull request by BrianDassow**

### CODE REVIEW

File: Game.gd [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
+		#get_tree().change_scene('res://ui/main-screen/MainScreen.tscn')
+				
 func createLevel():
 	var level = preload('res://levels/dungeon/dungeon.tscn').instance()
 	add_child(level)
 	
-func _on_player_disconnected(id):
-	print("someone disconnected")
-	$'/root/Game/dungeon/walls'.get_node(str(id)).queue_free()
+#func _on_player_disconnected(id):
```
Comment (MrLane13) [inline Game.gd]: Do all the comments need to stay?

File: characters/player/Player.gd [hunk 2]
```diff
@@ -62,6 +62,7 @@ func _physics_process(delta):
 #			set_position(player_position.linear_interpolate(destination,delta))
 		var current_velocity = get_position() - prev_position
 		
+		#Network.sendPositionAndVelocity(position, velocity)
```
Comment (MrLane13) [inline characters/player/Player.gd]: And here

File: characters/player/sprites/hero/Hero.gd [hunk 3]
```diff
@@ -4,6 +4,8 @@ func init(nickname, start_position, is_slave):
 	if !is_slave:
 		inventory = preload('res://characters/player/ui/inventory/Inventory.tscn').instance()
 		$'Sprite/Animation'.play('Stance W')
+	#var usernameLabel = Label.new()
```
Comment (MrLane13) [inline characters/player/sprites/hero/Hero.gd]: And here

File: global/Util.gd [hunk 4]
```diff
@@ -0,0 +1,13 @@
+extends Node
+
```
Comment (Ex3Dophos) [inline global/Util.gd]: Not a fan of calling this util. Especially when it looks like it only deals with changing screens. We can leave if or now though.

File: characters/player/Player.gd (hunk 2 above)
Comment (BrianDassow) [reply characters/player/Player.gd]: i dont want to change player.gd since we are refactoring

**Pull request merged**